src/frontend/js/types/Joanie.ts

```diff
@@ -170,13 +170,22 @@ export interface Enrollment {
   products: Product[];
 }
 
+// Contract
+
+export interface Contract {
+  contract_id: string;
+  learner_name: string;
+  product_title: string;
+  nb_signatures: number;
```

Comment:
WDYT `signatures_count`? It feels more fluent imo

Reply:
same same, i will use count if you prefer.